builder/azure/arm/config.go

```diff
@@ -79,6 +79,13 @@ type Config struct {
 	CaptureNamePrefix    string `mapstructure:"capture_name_prefix"`
 	CaptureContainerName string `mapstructure:"capture_container_name"`
 
+	// Shared Gallery
+	SharedGallerySubscription  string `mapstructure:"shared_gallery_subscription"`
```

Comment:
Please follow the same pattern as was used for plan_info.  Basically, add these elements to a struct.